library/std/src/sys/pal/unix/thread.rs

```diff
@@ -212,12 +217,33 @@ impl Thread {
         }
     }
 
+    #[cfg(target_os = "vxworks")]
+    pub fn set_name(name: &CStr) {
+        // FIXME: adding real STATUS type, OK, ERROR ... to libc crate eventually.
```

Comment:
If this type is only used for vxworks, I think we should just add it here, honestly.

Reply:
well because STATUS is just a c_int, and so on ... I was thinking they might serve later on.

Reply:
It looks like libc already has `OK` which is defined as a `c_int` https://github.com/rust-lang/libc/blob/0e28c864c25d2e9b0ab082947445efccef213da4/src/vxworks/mod.rs#L616. So I think `debug_assert_eq!(res, libc::OK);` should already work? We just discard the error condition so that isn't needed.

If a status type is used in other places, we should probably update libc first and then do the changes at once. Maybe just mark this `// FIXME(libc): ...` to make it more searchable.

Reply:
Cool I missed the OK.